contracts/tasks/runner/deployFull.ts

```diff
@@ -14,8 +14,9 @@ task("deploy-full", "Deploy environment")
   .addFlag("incremental", "Incremental deployment")
   .addFlag("strict", "Fail on warnings")
   .addFlag("verify", "Verify contracts at Etherscan")
+  .addParam("useQuadraticVoting", "Whether to use quadratic voting or not", true, types.boolean)
```

Comment:
And for deploy full and deploy poll you don't need it here as a param

Reply:
right makes sense